Automattic/vip-go-mu-plugins
# Allow to test all files with specific WP version

## Description

The bash script that we use to run unit tests locally would not work if the user wanted to test all files with a specific WordPress configuration. This PR addresses that and allows the user to input multiple optional parameters:

```
./bin/phpunit-docker.sh --file tests/path/to/test --version 5.4.4 --multisite 0
```

Also, we print more explicitly what options are being executed in each run.

Note that there's no need to deploy this PR, since it only contains changes to local development tooling.

## Changelog Description

### Allow to unit test all files with specific WP version

Updated internal tooling to better unit test the code.

## Checklist

Please make sure the items below have been covered before requesting a review:


## Steps to Test

1. Check out PR.
2. Run the testing script with multiple files/WP versions.
